index.js

```diff
@@ -35,3 +35,35 @@ exports.verifyPayment = function (platform, payment, cb) {
 		cb(null, result);
 	});
 };
+
+
+exports.cancelSubscription = function (platform, payment, cb) {
+	function syncError(error) {
+		process.nextTick(function () {
+			cb(error);
+		});
+	}
+
+	if (!payment) {
+		return syncError(new Error('No payment given'));
```

Comment:
This is absolutely not a requirement, but did you know you could write:

```js
return process.nextTick(cb, new Error('No payment given'));
```

It's not much longer, and you don't really need that `syncError` helper function. Just a suggestion, not a blocker.

Reply:
a above (https://github.com/Wizcorp/node-iap/pull/55/files#r218949991)

Reply:
Got it, in that case let's land this first :)
Follow-up PR always appreciated of course! :)